mundus/interim_report_2023_aug_22.md

```diff
@@ -0,0 +1,351 @@
+
+# <a name="deployment"></a>Deployment check: storage
+
+We thoroughly examine both public and private storage, as well as immutable and constant variables, to ensure that there are no misconfigurations, especially:
+
+1. Incorrect or outdated addresses to other smart contracts referenced in the scope of work (SoW) - this includes addresses stored in variables, mappings, and other data structures.
+2. Any references to other smart contracts or externally owned accounts (EOAs) that may be incorrect or outdated.
+3. Any incorrect protocol settings stored in variables or other data structures.
+4. Misconfigurations related to the roles and permissions of the contract.
+5. Governance issues that may impact the operation and business logic of the smart contract.
+
+## 1. AaveOracle
+
+| issue # | issue<br />type        |
+| ------- | ---------------------- |
+| 1       | incorrect value        |
+| 2       | incorrect value        |
+| 3       | out of scope contracts |
+
+1. `fallbackOracle` = `0x0`
+2. `baseCurrency` = `0x0`
+3. `assetsSources`, `mapping(address => AggregatorInterface)`
+
+**NOTE:** out of scope contracts are marked with <span class="hm">&#9888;</span>.
+
+| asset                                                                                                                                      | source                                                                                                                                     |
+| ------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| <span class="hm">&#9888;</span> [0x90059C32Eeeb1A2aa135...](https://explorer.zksync.io/address/0x90059C32Eeeb1A2aa1351a58860d98855f3655aD) | [0x1E3720185512d22C7352...](https://explorer.zksync.io/address/0x1E3720185512d22C7352759e79dC3515d752AA50) <span class="hm">&#9888;</span> |
+| <span class="hm">&#9888;</span> [0x75Af292c1c9a37b3EA2E...](https://explorer.zksync.io/address/0x75Af292c1c9a37b3EA2E6041168B4E48875b9ED5) | [0x3D5BcB12800A092FC85C...](https://explorer.zksync.io/address/0x3D5BcB12800A092FC85Ca00837594146F274C273) <span class="hm">&#9888;</span> |
+| <span class="hm">&#9888;</span> [0x2039bb4116B4EFc145Ec...](https://explorer.zksync.io/address/0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181) | [0x1a963D0C6bF364C1C8AE...](https://explorer.zksync.io/address/0x1a963D0C6bF364C1C8AE4F17b6aB773c627cEFB7) <span class="hm">&#9888;</span> |
+| <span class="hm">&#9888;</span> [0xFD282F16a64c6D304aC0...](https://explorer.zksync.io/address/0xFD282F16a64c6D304aC05d1A58Da15bed0467c71) | [0xCd16A63d1960Afe718c4...](https://explorer.zksync.io/address/0xCd16A63d1960Afe718c4FE62D0b8D8f19Fc29618) <span class="hm">&#9888;</span> |
+
```

Comment:
Ideally 3rd party contracts like USDT, USDC etc.. Don't need to be considered "Out-of-Scope" in our opinion. 

But oracles (second column) that are out of scope, we are happy to redeploy them to match the in-scope contracts. Please let us know if this is suitable.